flyteidl/protos/flyteidl/core/execution.proto

```diff
@@ -89,6 +89,7 @@ message TaskLog {
     string name = 2;
     MessageFormat message_format = 3;
     google.protobuf.Duration ttl = 4;
+    bool ShowWhilePending = 5;
 }
```

Comment:
It might also be interesting to hide certain log links again once the task finished, e.g. for "flyin" tasks as the VSCode Server isn't reachable anymore and the link is dead.